Package@swift-4.2.swift

```diff
@@ -0,0 +1,21 @@
+// swift-tools-version:4.2
+import PackageDescription
+
+let package = Package(
+    name: "Nimble",
+    products: [
+        .library(name: "Nimble", targets: ["Nimble"]),
+    ],
+    targets: [
+        .target(
+            name: "Nimble", 
+            dependencies: []
+        ),
+        .testTarget(
+            name: "NimbleTests", 
+            dependencies: ["Nimble"], 
+            exclude: ["objc"]
+        ),
```

Comment:
@ikesyo the original `Package.swift` file have those trailing commas.
Also, I think it's a good think to keep trailing commas in case if you need to add a new item later the diff will have one less line.

Reply:
Sure, it's okay for ignoring the warning for now.